core/src/main/java/tc/oc/pgm/scoreboard/SidebarMatchModule.java

```diff
@@ -481,22 +481,17 @@ private void renderSidebar() {
           boolean horizontalCompact =
               MAX_LENGTH < (3 * sortedWools.size()) + (3 * (sortedWools.size() - 1)) + 1;
           TextComponent.Builder woolText = text();
-          if (!horizontalCompact) {
-            // If there is extra room, add another space to the left of the wools to make them
-            // appear more centered.
-            woolText.append(space());
-          }
-
           for (Goal<?> goal : sortedWools) {
             if (goal instanceof MonumentWool && goal.hasShowOption(ShowOption.SHOW_SIDEBAR)) {
               MonumentWool wool = (MonumentWool) goal;
-              woolText.append(space());
+              TextComponent spacer = space();
               if (!firstWool && !horizontalCompact) {
-                woolText.append(space()).append(space());
+                spacer = spacer.append(space()).append(space());
```

Comment:
Maybe weird suggestion but since the sidebar stuff might get rendered a lot maybe we can create the second spacer statically? `space()` itself does not create any objects, but the triple space could be created once.

```java
Component spacer = (!firstWool && !horizontalCompact) ? TRIPLE_SPACE : space()
```

You could go a step further and just check for this when you are appending the rendered text(To not create the reference at all), but I think that will be harder to read than you gain in performance I imagine...